WcaOnRails/app/views/registrations/register.html.erb

```diff
@@ -6,6 +6,56 @@
     <p><b><%= t('registrations.registration_requirements') %></b>
     </p>
     <div>
+      <% if @competition.use_wca_registration? %>
```

Comment:
Is this all a copy of the code you added to WcaOnRails/app/views/competitions/_competition_info.html.erb? This is a lot of logic, can you pull this out into a partial instead?

Reply:
That makes total sense, I will do it!